### PR TITLE
feat: add support for GENERATE_UUID

### PR DESCRIPTION
**Pull request by taka-oyama**
Fix: #173 

## Checklist

- [x] CHANGELOG

### CODE REVIEW

File: src/Schema/ColumnDefinition.php [hunk 1]
```diff
@@ -0,0 +1,14 @@
+<?php
+
+namespace Colopl\Spanner\Schema;
+
+use Illuminate\Database\Query\Expression;
+use Illuminate\Database\Schema\ColumnDefinition as BaseColumnDefinition;
+
+class ColumnDefinition extends BaseColumnDefinition
+{
+    public function generateUuid(): static
```
Comment (tyabu12) [inline src/Schema/ColumnDefinition.php:10]:
```suggestion
    /*
     * @return $this
     */
    public function generateUuid(): static
```

**Review by tyabu12 — CHANGES_REQUESTED** [review]

File: src/Schema/Blueprint.php [hunk 2]
```diff
@@ -94,7 +95,16 @@ public function tinyIncrements($column)
...
     }
 
-    // region Spanner Specific Types
+    /**
+     * @inheritDoc
+     * @return SpannerColumnDefinition
+     */
+    public function uuid($column = 'uuid')
+    {
+        $definition = new SpannerColumnDefinition(['type' => 'uuid', 'name' => $column]);
```
Comment (tyabu12) [inline src/Schema/Blueprint.php:104]: How about overriding `addColumn()` method for consistency?

Comment (taka-oyama) [reply src/Schema/Blueprint.php]: That would make it more consistent but less robust. `generate_uuid` should only be available when a column is defined as uuid. I've renamed `ColumnDefinition` to `UuidColumnDefinition` for clarity.

**Review by tyabu12 — APPROVED** [review]
LGTM

**Comment by taka-oyama** [comment]
Thanks for the review!

**Pull request merged**